helm/micro-integrator/values.yaml

```diff
@@ -0,0 +1,84 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+wso2:
+  # WSO2 subscription parameters. If you do not have an active subscription, do not provide values for the parameters.
+  # If you do not possess an active WSO2 subscription already, you can sign up for a WSO2 Free Trial Subscription at (https://wso2.com/free-trial-subscription).
+  subscription:
+    username: ""
+    password: ""
+  deployment:
+    wso2microIntegrator:
+      # If a custom image must be used, uncomment 'dockerRegistry' and provide its value.
+#      dockerRegistry: ""
+      imageName: "wso2mi"
+      imageTag: "1.2.0"
+      # Number of deployment replicas
+      replicas: 1
+      # Minimum number of seconds for which a newly created pod should be ready without any of its containers crashing, for it to be considered available.
+      minReadySeconds: 30
+      strategy:
+        rollingUpdate:
+          # The maximum number of pods that can be scheduled above the desired number of pods.
+          maxSurge: 1
+          # The maximum number of pods that can be unavailable during the update.
+          maxUnavailable: 0
+      # Indicates whether the container is running.
+      livenessProbe:
+        # Number of seconds after the container has started before liveness probes are initiated.
+        initialDelaySeconds: 35
+        # How often (in seconds) to perform the probe.
+        periodSeconds: 10
+      # Indicates whether the container is ready to service requests.
+      readinessProbe:
+        # Number of seconds after the container has started before readiness probes are initiated.
+        initialDelaySeconds: 35
+        # How often (in seconds) to perform the probe.
+        periodSeconds: 10
+      # These are the minimum resource recommendations for running WSO2 Micro Integrator
+      resources:
+        requests:
+          # The minimum amount of memory that should be allocated for a Pod
+          memory: "512Mi"
+          # The minimum amount of CPU that should be allocated for a Pod
+          cpu: "500m"
+        limits:
+          # The maximum amount of memory that should be allocated for a Pod
+          memory: "1Gi"
+          # The maximum amount of CPU that should be allocated for a Pod
+          cpu: "1000m"
+      # Environment variables for the Micro integrator deployment.
+      envs:
+  # Configure Ingress for the Micro integrator deployment.
+  ingress:
+    # Hostname for the Micro Integrator.
+    host:
+    # Annotations for the Micro Integrator Ingress.
+    annotations:
+      ingress.kubernetes.io/ssl-redirect: 'true'
+      kubernetes.io/ingress.class: "nginx"
+      nginx.ingress.kubernetes.io/affinity: "cookie"
```

Comment:
@ThilinaManamgoda just out of curiosity, do we need session affinity in Micro Integrator?

Reply:
It is not needed. I'll remove this.